package.json

```diff
@@ -96,6 +100,7 @@
     "prettier": "^2.0.5",
     "react-test-renderer": "^16.12.0",
     "rimraf": "^3.0.2",
+    "scheduler": "^0.18.0",
```

Comment:
What's `scheduler` for?

Reply:
It's used in conjunction with `React.Profiler` to trace updates. See: https://gist.github.com/bvaughn/8de925562903afd2e7a12554adcdda16